libafl_qemu/src/helpers/snapshot.rs

```diff
@@ -73,6 +73,12 @@ pub struct MappingInfo {
     pub size: usize,
 }
 
+pub enum IntervalSnapshotFilter {
```

Comment:
I'm not sure if this is something we should expose like this.
It's great to have this around for debugging or dirty fix as last resort, but I'm scared people rely too much on this instead of reporting bugs in the snapshot systems.
Also, it would be nice to have this work generically with the `IsSnapshotManager` maybe? I didn't implement the trait for usermode snapshot yet, but I guess it could be the right moment to do so.

Reply:
Or maybe we could emit a warning when using this asking to issue a bug report or something similar?

Reply:
You could add a `#[deprecated]` flag that will show at compile time? (But feels a bit funny to add a new API as deprecated)

Reply:
I doesn't seem there is another way to issue warnings at compile time in stable, I guess it's a good alternative